LiskHQ/lisk
# Logs are created dynamically based on network name - Closes #2452

### What was the problem?

Logs were not unique to each network.

### How did I fix it?

Based on PR feedback I added the full path to each network. When creating a custom network the paths need to be set on its config file.

### How to test it?

- Run  `node app.js` and it should create `logs/devnet` (default network name)
- Create a config, set the log paths for both properties logFile and db.logFile and run `node app.js -n <your_network_name>` and it should create the logs in `logs/<your_network_name> ` 


### Review checklist

* The PR resolves #2452
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
